tests/rbd/rbd_snap_clone_imported_image.py

```diff
@@ -57,7 +57,7 @@ def test_snap_clone(rbd, pool_type, **kw):
 
     finally:
         if not kw.get("config").get("do_not_cleanup_pool"):
-            rbd.clean_up(pools=[pool])
+            rbd.clean_up(pools=[pool], flag="all")
```

Comment:
then we should add this way for all other existing tests too,
can you think some other solution?

Reply:
Does it ok without `flag` as well, 
anyway we have to delete the pools that were created during that test 
@sunilkumarn417 Any thoughts?

Reply:
Pass logs:  http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-AZMF2S

Reply:
thanks @aarsharm this solution looks good to me,
but can you add this way for all other existing tests where ever EC pools exist.
bcz this is essential for bare-metal runs

Reply:
Sure will check and add

Reply:
ok